src/platform_layers/simulator_platform_layer/src/uhttp_downloader.cpp

```diff
@@ -1,440 +0,0 @@
-/**
```

Comment:
should we move this to cotnent_downloader and remove the curl downloader?  I like this (again, I'm biased) as it has *no* external dependencies.  It will work cross platform and doesn't need to spawn curl.

Reply:
We could move this (later), but I'd rather keep curl downloader also. Let customer decide what to use.